types/index.d.ts

```diff
@@ -45,7 +45,7 @@ export declare class Store<S> {
 
 export function createStore<S>(options: StoreOptions<S>): Store<S>;
 
-export function useStore<S = any>(): Store<S>;
+export function useStore<S = any>(injectKey?: InjectionKey<Store<any>> | string): Store<S>;
```

Comment:
I think it should be ` InjectionKey<Store<S>>`?

```js
export function useStore<S = any>(injectKey?: InjectionKey<Store<S>> | string): Store<S>;
```

Reply:
😳